11ty/definitions/hispanic.md

```diff
@@ -1,5 +1,13 @@
 ---
 title: Hispanic
 slug: hispanic
-defined: false
----
\ No newline at end of file
+defined: true
+reading:
+    - text: "Remezcla: Hispanic vs. Latino vs. Latinx: A Brief History of How These Words Originated"
+      href: https://remezcla.com/features/culture/latino-vs-hispanic-vs-latinx-how-these-words-originated/
+---
+
+of Spanish, or Spanish-speaking origin. 
```

Comment:
Add here: It is considered a pan-ethnic term and does not provide context for someone's race or country of origin.

Reply:
I've updated this definition, along with Latinx to call out the pan-ethnic nature of this term